jmason87/ygs-watch-fe
# Jm seasons crud

## What does this PR do?
This PR adds create functionality for seasons
## Related PRs

## Testing Steps
- navigate to http://localhost:8080/hockey/season/addSeasons
- fill out form and see that form clears upon submission
- see that new season record was added to db 
- see that the new season is displayed on the player table for the payer you selected
## Checklist before merging
